Fluid/Filters/NumberFilters.cs

```diff
@@ -19,6 +22,43 @@ public static FilterCollection WithNumberFilters(this FilterCollection filters)
             return filters;
         }
 
+        public static FluidValue Abs(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            return NumberValue.Create(Math.Abs(input.ToNumberValue()), true);
+        }
+
+        public static FluidValue AtLeast(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            var first = arguments.At(0);
+            double num = first.ToNumberValue();
+
+            if (first is NumberValue number)
```

Comment:
It doesn't need to instantiate any number. Just "compare" the two ToNumber values and return whichever argument is valid.